### ISSUE TITLE
As a user, moving/copying files between zones erases metadata

### ISSUE THREAD

**Issue by ganqqwerty** [INCF/ids-tools#18]
I use iDrop Desktop. 

Steps to reproduce the bug: 
1) I uploaded the file to my personal folder (incf/home/katkov)
2) I added the metadata to this file: 10 fieleds of various datatypes
3) I click the button Copy/Move and move it to another zone. Particularly I moved it to the folder bbp/home/public . 
4) I see that all the metadata have gone from the file.

**Comment by cansmith** [comment]
This is just the way iRODS works in a federation. The ids-tools command-line utility 'ids-copy-dataset' was implemented to work around this issue.

**Comment by ganqqwerty** [comment]
Moving and copying files between while preserving metadata are the key features for the data sharing platform. For instance moving the file to the public folder is the simplest way to share the file. Can you estimate how much time and resources does it takes to include this feature to either iDrop Web or iDrop Desktop?

**Comment by brainstorm** [comment]
Sorry, I mixed my answer between tickets, @michael-conway and @ganqqwerty.

As we mentioned on the skype meeting, we need this metadata being erased cross-zones to be solved:

1) Upstream, in core iRODS, not as an independent `ids-tool`.
2) Validate the behaviour through the iDROP UI's (checking the metadata after data has moved between zones).

Reopening...

**Comment by ganqqwerty** [comment]
https://groups.google.com/forum/#!topic/iROD-Chat/ZnHvOiyRIzg - here is the discussion

**Comment by michael-conway** [comment]
Yes, on the way in I actually had the same thought.  It's really a code
smell, and likely should be optionally or by default the behavior of the
copy and move protocols themselves.

What I may end up doing is 'overheading' in jargon for versions prior to
any potential patch and triggering that by detecting the iRODS version.
 I'm gong to send a GitHub issue to the core iRODS server folks on this!

Cheers
MC

**Comment by brainstorm** [comment]
Being handled upstream on https://github.com/DICE-UNC/jargon/issues/25.

Closing.

**Comment by ganqqwerty** [comment]
What is the current practice of closing tickets in this tracker? I mean, the fact that we have passed the task to DICE doesn't mean that the problem is solved and doesn't exist anymore, right? I propose to wait for the solution and keep the ticket opened.

**Comment by michael-conway** [comment]
Hi all, I'd keep it open and I can put in an update when I hear back. In
the meantime I think it's a decent addition to add 'copy metadata'
functionality in jargon to read avus and propagate to another target.  That
won't take too long, and is generally useful.

I was thinking about adding an option in jargon to make that happen on
move/copy and this would make the behavior consistent for prior versions of
iRODS as well.
